lib/dl_db_testing/dl_db_testing/database/base.py

```diff
@@ -158,15 +158,22 @@ class DbTableBase:
     def name(self) -> str:
         return self.table.name
 
-    def insert(self, data: Union[dict, List[dict]]) -> None:
+    def insert(self, data: Union[dict, List[dict]], bulk_insert: bool = False, chunk_size: int = 200) -> None:
         # TODO: Use insert_into_table
         if not self.can_insert:
             raise RuntimeError("Can't insert into table")
         if isinstance(data, dict):
             self.db.execute(self.table.insert(data))
         elif isinstance(data, list):
-            for row in data:
-                self.insert(row)
+            if bulk_insert:
+                rest = data
+                while rest:
```

Comment:
`for chunk in itertools.batched(data, chunk_size):`

Reply:
New in version 3.12.
I know about it and have been waiting for it for years, but we can't use it yet :(

Reply:
oh, my bad, i missed that
can you please add a TODO so we won't forget to use it after upgrading to 3.12?